### DISCUSSION TITLE
Animated back to top button with CSS

### DISCUSSION THREAD

**Discussion by LITUATUI** [Lednerb/bilberry-hugo-theme#487] [Ideas]
The idea is to show a back to top button only after scrolling with CSS.

[This](https://www.freecodecamp.org/news/css-only-back-to-top-button/
) article shows some methods that only use CSS, no JS.

I especially like [this](https://codepen.io/t_afif/pen/ExwbZKQ/86f79c96520a0c864324db70a6b76c76) method with a fading button.

What do you guys think?

**Comment by igor-baiborodine** [comment]
@LITUATUI, it would be nice to have this back-to-top button. I recently saw it on https://bluesoulceviriekibi.net/, which uses a customized version of the Bilberry Hugo theme. If you would like to implement this feature, please go ahead. We'll be very grateful to you.
